source/accessible-media/production-captions.html.erb.md

```diff
@@ -89,35 +94,58 @@ formatted and timestamped.
 
 ### Caption quality
 
-Always provide the highest-quality captions that convey 100% accuracy.
+{::nomarkup}
+<%= ref :start %>
+{:/nomarkup}
+
 See [resources for writing quality captions](#resources-for-writing-quality-captions) for more
-information. Keep the following basic points in mind when writing
+information.
+
+{::nomarkup}
+<%= ref :middle %>
+{:/nomarkup}
+
+Always provide the highest-quality captions that convey 100% accuracy.
+Keep the following points in mind when writing
 captions:
 
 -   Ensure that there are no spelling errors. This includes the names of
     characters or speakers.
 -   Use conventional grammar rules. After end punctuation (period,
-    question mark, exclamation point, etc.), always begin a new caption.
+    question mark, exclamation point, etc.), always begin a new caption block.
 -   Do not edit unless you have a specific reason to do so (e.g., to
     achieve a specific reading level). Fillers such as "um," "ah," etc.,
     can be deleted to save reading time unless doing so alters the
     representation of the speaker.
 -   Do not censor: the captions should always accurately represent what
```

Comment:
What if censoring is required (by law)? And what if the dialog in the video is censored?